### ISSUE TITLE
Outstanding bills not displayed on Bill page 

### ISSUE THREAD

**Issue by mimorita** [nadineproject/nadine#428]
At /staff/billing/bills/outstanding/

Some outstanding bills are not showing. The two that I noticed are Bill 17082 and Bill 17073. Both should show in the Subscriptions Due section.

**Comment by jsayles** [comment]
Yes, this is a pretty serious bug.  The issue is that at one point we changed up the billing page to rely on hidden fields that keep track of the bill amount.  For some reason on these two those fields were not calculated correctly.  That is the bug although I don't see anything here to tell me why this was.

The work-around is to go to the bill -> admin -> save.  You don't need to make any changes, just open the bill in the admin console and save it.  This triggers the hidden fields to be recalculated.  The Recalculated button does something else.

Both of those bugs are now showing but this bug is not closed until the underlying cause  is fixed.

**Comment by mimorita** [comment]
Ok, thanks. Is there a way to discover what bills might not be displayed, but should be? I am afraid of missing them.

**Comment by jsayles** [comment]
I recalculated all the caches for all the bills over all the time and 3 more bills popped up:  17015, 14284, 17030 totaling $130 in lost revenue from this bug.  So that at least shows us this isn't an isolated case but also isn't too damaging.
